backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -158,11 +159,12 @@ object VeloxBackendSettings extends BackendSettingsApi {
               CharVarcharUtils.getRawTypeString(metadata) + " not support"
             case StructField(_, TimestampType, _, _) => "TimestampType not support"
           }
-          if (!GlutenConfig.getConf.forceComplexTypeScanFallbackEnabled) {
-            validateTypes(typeValidator)
-          } else {
-            validateTypes(orcTypeValidatorWithComplexTypeFallback)
-          }
+          ValidationResult.succeeded
```

Comment:
@kecookier have you use complex datatype in ORC format?

Reply:
This is for just a try how the change affect ORC related UTs. It turns out timestamp support has result mismatch issue. We may still keep this option for ORC as it's not fully verified as Parquet. For parquet, we have problems in nested struct type support in Gluten and I am looking into that.

Reply:
> @kecookier have you use complex datatype in ORC format?

Yes, we disable option `forceComplexTypeScanFallbackEnabled`.

Reply:
Does the timestamp support result match issue involves Velox? Could we create a new issue in Velox to track it?

Reply:
Issue [6831](https://github.com/apache/incubator-gluten/issues/6831) is created for track.

Reply:
Thanks~